tests/test_calendar.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+from django.core.urlresolvers import reverse
+
+from councilmatic_core.models import Event
+from nyc.utils import create_ics_output
+  
+@pytest.mark.django_db
+def test_ics_output():
+  event = Event.objects.all().first()
+  ics_content = create_ics_output(event).decode("utf-8")
+
+  # The content should include the event name.
+  assert (event.name in ics_content) == True 
+
+@pytest.mark.django_db
+def test_ical_export(django_db_setup, client):
+    event = Event.objects.all().first()
+    url = reverse('nyc:ical_export', kwargs={'slug': event.slug})
+    response = client.get(url) 
+
+    # The response should return a 200, present a content type of 'text/calendar', and contain an attachment with the slug as its file name.
+    assert response.status_code == 200
+    assert response['Content-Type'] == 'text/calendar'
+    assert response['Content-Disposition'] == 'attachment; filename={}.ics'.format(event.slug)
+
+@pytest.mark.django_db
+def test_google_calendar_export(django_db_setup, client):
+    event = Event.objects.all().first()
+    url = reverse('nyc:google_calendar_export', kwargs={'slug': event.slug})
+    response = client.get(url) 
+
```

Comment:
I'd suggest mocking `google_calendar_export_helper` and testing that it's called with the `event` object as an argument when you GET the `google_calendar_export` route. 

This will involve using `pytest-mock` (`pip install...`), including the `mocker` fixture in the test, and doing something like:

```python
mock_gcal_helper = mocker.patch('utils.google_calendar_export_helper', autospec=True)

# GET the route that should call the method

_, call_args = mock_gcal_helper.call_args

assert event in call_args
```

I'm freestyling so the call args thing may be wonky, but I think that's about right!